src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+iexApiToken = ${IEX_API_TOKEN}
```

Comment:
is there no way to put this in the yml?

Reply:
I just tried doing it in yml and it works. It is the exact same way.

Reply:
> is there no way to put this in the yml?
I just tried doing it in yml and it works. It is the exact same way.

Reply:
lets do that instead then. that way we only have one config file

Reply:
Make the changes and added it to the branch.